tests/typing_tests/.gitignore

```diff
@@ -0,0 +1 @@
+*_numpy.pyi
```

Comment:
Question: Is it possible to generate files under the temporary directory? Or will that invalidate mypy cache?